velox/functions/sparksql/UuidFunction.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <boost/uuid/uuid.hpp>
+#include <boost/uuid/uuid_io.hpp>
+
+#include "velox/core/QueryConfig.h"
+#include "velox/functions/Macros.h"
+
+namespace facebook::velox::functions::sparksql {
+
+template <typename T>
+struct UuidFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  static constexpr bool is_deterministic = false;
+
+  // Spark would set the seed with 'random.nextLong()' by 'ResolveRandomSeed'
+  // rule.
+  void initialize(const core::QueryConfig& config, const int64_t* seedInput) {
+    int64_t seed = seedInput ? *seedInput : 0L;
```

Comment:
In `initialize()`, seeInput is not nullptr means it is constant input. So maybe, just throws exception for non-constant input instead of using 0L.
(it's different from `callNullable` where nullptr arg means NULL input)

Reply:
Thanks, add the check.